procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -0,0 +1,631 @@
+{
```

Comment:
<p>Same as before.</p><p> </p><p>Consider making a markdown section in a cell above:</p><p> </p><pre class="ql-syntax" spellcheck="false">## Take an engineering test frame (analogous to a science frame) and make sure it lands
</pre><p> </p><p>instead of <code>tmp</code> use a more descriptive name, e.g.; <code>image_in_oods = ...</code>.</p><p> </p><p>Instead of <code>if ... : raise</code></p><p> </p><pre class="ql-syntax" spellcheck="false">assert tmp.statusCode != 0, "Ingestion was not successful!"
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>